cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -661,15 +667,15 @@ defaultPartialNodeConfiguration =
         }
     , pncDeadlineTargetOfRootPeers        = Last (Just deadlineRoots)
     , pncDeadlineTargetOfKnownPeers       = Last (Just deadlineKnown)
-    , pncDeadlineTargetOfEstablishedPeers = Last (Just deadlineEstablished)
+    , pncDeadlineTargetOfEstablishedPeers = Last (Just 30) -- ^ TODO reset to deadlineEstablished for next o-n-release
     , pncDeadlineTargetOfActivePeers      = Last (Just deadlineActive)
     , pncChainSyncIdleTimeout           = mempty
     , pncDeadlineTargetOfKnownBigLedgerPeers       = Last (Just deadlineBigKnown)
     , pncDeadlineTargetOfEstablishedBigLedgerPeers = Last (Just deadlineBigEst)
     , pncDeadlineTargetOfActiveBigLedgerPeers      = Last (Just deadlineBigAct)
     , pncSyncTargetOfActivePeers        = Last (Just syncActive)
     , pncSyncTargetOfKnownBigLedgerPeers       = Last (Just syncBigKnown)
-    , pncSyncTargetOfEstablishedBigLedgerPeers = Last (Just syncBigEst)
+    , pncSyncTargetOfEstablishedBigLedgerPeers = Last (Just 50) -- ^ TODO reset to syncBigEst for next o-n-release
```

Comment:
I believe this should be `(Just 40)` to match the `SyncTargetNumberOfEstablishedBigLedgerPeers` value change to 40.